### ISSUE TITLE
`observeAskingTargets` not detecting "slewing at tracking speed"

### ISSUE THREAD

**Issue by EastEriq** [PolishookDavid/LAST_OCS#51]
self note.

Seen on M3, looping on discovering that the unit was not ready (mount kept being "slewing"), attempting `checkWholeUnit`, getting a new target and failing again.

Check `matlab_03_master_20241127000531_stdout.log`.
Also note that during the checks the cameras report no status.

**Comment by EastEriq** [comment]
In the log, the typical "get target - fail" sequence looks like:
```
{20:49:18.478|obs.unitCS[03]} Target acquired: "906", 20 exposures x 20 sec
{20:49:25.004|obs.unitCS[03]} Mount status is "slewing"
{20:49:25.177|obs.unitCS[03]} Mount status is "slewing"
{20:49:25.184|obs.unitCS[03]} Mount is not OK after 2 retries
{20:49:25.189|obs.unitCS[03]} Unit is not ready for the next exposure
{20:49:25.196|obs.unitCS[03]} Unit didn't finish visit exposures
{20:49:25.396|obs.unitCS[03]} Camera 1 acquired 0/20 images
{20:49:25.597|obs.unitCS[03]} Camera 2 acquired 0/20 images
{20:49:25.735|obs.unitCS[03]} Camera 3 acquired 0/20 images
{20:49:25.910|obs.unitCS[03]} Camera 4 acquired 0/20 images
{20:49:25.919|obs.unitCS[03]} Camera 1 status is "", sending camera.abort
{20:49:26.788|obs.unitCS[03]} Camera 2 status is "", sending camera.abort
{20:49:27.564|obs.unitCS[03]} Camera 3 status is "", sending camera.abort
{20:49:28.460|obs.unitCS[03]} Camera 4 status is "", sending camera.abort
{20:49:29.297|obs.unitCS[03]} Mount stopped tracking, aborting visit!
{20:49:29.302|obs.unitCS[03]} Checking definitions and connections of unit 03:
{20:49:29.448|obs.unitCS[03]} Slave 1 status: "alive"
{20:49:29.597|obs.unitCS[03]} Slave 2 status: "alive"
{20:49:29.688|obs.unitCS[03]} Slave 3 status: "alive"
{20:49:29.820|obs.unitCS[03]} Slave 4 status: "alive"
{20:49:31.126|obs.unitCS[03]} mount is powered
{20:49:31.883|obs.unitCS[03]} checking status of camera 1
{20:49:32.057|obs.unitCS[03]} camera 1 is idle, good
{20:49:33.475|obs.unitCS[03]} checking status of camera 2
{20:49:33.639|obs.unitCS[03]} camera 2 is idle, good
{20:49:36.163|obs.unitCS[03]} checking status of camera 3
{20:49:36.352|obs.unitCS[03]} camera 3 is idle, good
{20:49:37.653|obs.unitCS[03]} checking status of camera 4
{20:49:37.755|obs.unitCS[03]} camera 4 is idle, good
{20:49:39.155|obs.unitCS[03]} focuser 1 check passed
{20:49:39.987|obs.unitCS[03]} focuser 2 check passed
{20:49:40.719|obs.unitCS[03]} focuser 3 check passed
{20:49:41.459|obs.unitCS[03]} focuser 4 check passed
{20:49:41.469|obs.unitCS[03]} all checks OK
{20:49:41.472|obs.unitCS[03]}   Properties of the last images:
{20:49:41.647|obs.unitCS[03]}   Camera 1: FWHM 2.813
{20:49:41.777|obs.unitCS[03]}   Camera 2: FWHM 2.969
{20:49:41.941|obs.unitCS[03]}   Camera 3: FWHM 2.969
{20:49:42.096|obs.unitCS[03]}   Camera 4: FWHM 2.813
```
The empty camera stati are explained because of the early return of  `unitCS.readyToExpose`, which doesn't bother to check camera status because it failed on the mount. `observeAskingTargets` just reports what it gets from `.readyToExpose` and does not check further.

**Comment by EastEriq** [comment]
(I unfortunately deleted and lost the content I was going to paste)

No, the problem is very likely overcurrent. I did a FullTrackingTest and got stuck already at the third target. Inspection showed that HA was not moving at all, `Mount.Status` kept being 'slewing', `Mount.SlewingSpeed` was [12 12], but `Mount.statusReport` had many flags on, including HA current limited. And indeed HA current was 4.5A.

I then deactivated and parked the mount.
